bootstrap.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Source this file to give an environment for running end to end tests
+set -o
+set -e
+
+sudo yum install -y \
+	make \
+	wget \
+	curl \
+	llvm \
+	tar \
+	unzip \
+	git \
+	patch \
+	gcc
+
+curl -L https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer | bash
```

Comment:
This is specifically for the Vagrant VM. I will make that clear in a comment / by renaming the file.